frontend-maven-plugin/src/main/java/com/github/eirslett/maven/plugins/frontend/mojo/AbstractFrontendMojo.java

```diff
@@ -65,6 +65,12 @@ public abstract class AbstractFrontendMojo extends AbstractMojo {
     @Parameter(defaultValue = "${repositorySystemSession}", readonly = true)
     private RepositorySystemSession repositorySystemSession;
 
+    /**
+     * Enables node installation using node version manager
+     */
+    @Parameter(property = "useNodeVersionManager", defaultValue = "true")
+    private boolean useNodeVersionManager;
```

Comment:
Can we do one more like you suggested before? Just direct path to where node binary is. Both Maven property and use `System.getEnv`. I'm worried that someone will use an old branch with old version of this in the future and it won't support new version of FNM/NVS/etc. If we can tell them "no worries" modify your `.mvn/cnfig` and/or shell envinroment, that will be a great get out of jail pass.

Reply:
Added `installedNodeDirectory` property and `AFMP_INSTALLED_NODE_DIRECTORY` env var for specifying node explicitly